extensions/oidc-client-registration/runtime/src/main/java/io/quarkus/oidc/client/registration/runtime/OidcClientConfigurationException.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.oidc.client.registration.runtime;
+
+@SuppressWarnings("serial")
+public class OidcClientConfigurationException extends RuntimeException {
```

Comment:
Javadoc missing, when is this exception thrown?

Reply:
It is a copy and paste side-effect :-), this exception class has gone, thanks for spotting it